state/core_access.go

```diff
@@ -367,8 +355,21 @@ func (ca *CoreAccessor) BalanceForAddress(ctx context.Context, addr Address) (*B
 		return nil, fmt.Errorf("cannot convert %s into sdktypes.Int", string(value))
 	}
 	// verify balance
+	var proofOps *crypto.ProofOps
+	if result.GetProofOps() != nil {
+		proofOps = &crypto.ProofOps{
+			Ops: make([]crypto.ProofOp, len(result.ProofOps.Ops)),
+		}
+		for i, proofOp := range result.ProofOps.Ops {
+			proofOps.Ops[i] = crypto.ProofOp{
+				Type: proofOp.Type,
+				Key:  proofOp.Key,
+				Data: proofOp.Data,
+			}
+		}
+	}
 	err = ca.prt.VerifyValueFromKeys(
-		result.Response.GetProofOps(),
+		proofOps,
```

Comment:
If `proofOps == nil` (possible with the if check above), how does this method behaves?